src/main/java/cc/oakk/ladder/controller/LadderGame.java

```diff
@@ -1,48 +1,92 @@
 package cc.oakk.ladder.controller;
 
 import cc.oakk.ladder.model.ladder.Ladder;
-import cc.oakk.ladder.model.Person;
+import cc.oakk.ladder.model.person.Person;
+import cc.oakk.ladder.model.person.Persons;
+import cc.oakk.ladder.model.result.Result;
+import cc.oakk.ladder.model.result.Results;
 import cc.oakk.ladder.model.size.LadderHeight;
+import cc.oakk.ladder.model.trace.Trace;
+import cc.oakk.ladder.model.trace.dto.TraceResultsDto;
 import cc.oakk.ladder.util.RandomUtils;
-import cc.oakk.ladder.util.ValidationUtils;
 import cc.oakk.ladder.view.InputView;
+import cc.oakk.ladder.view.QueryView;
 import cc.oakk.ladder.view.ResultView;
 
 import java.util.Arrays;
-import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
+import static cc.oakk.ladder.util.ValidationUtils.tryUntilSuccess;
+
 public class LadderGame {
+    public static final Person ALL = new Person("all");
+
     private final InputView inputView;
     private final ResultView resultView;
+    private final QueryView queryView;
 
-    public LadderGame(InputView inputView, ResultView resultView) {
+    public LadderGame(InputView inputView, ResultView resultView, QueryView queryView) {
         this.inputView = inputView;
         this.resultView = resultView;
+        this.queryView = queryView;
     }
 
     public void start() {
-        List<Person> persons = ValidationUtils.tryUntilSuccess(() -> Arrays.stream(inputView.getPersonNames())
-                .map(Person::new)
-                .collect(Collectors.toList()), inputView::printError);
-
-        LadderHeight height = ValidationUtils.tryUntilSuccess(() -> new LadderHeight(inputView.getLadderHeight()),
-                inputView::printError);
+        Persons persons = getPersons();
+        Results results = getResults(persons);
+        LadderHeight height = getHeight();
 
         Ladder ladder = new Ladder(persons.size(), height.get());
         initLadder(ladder);
```

Comment:
초기화라는 네이밍은 이전에 이미 Ladder 객체 생성시 초기화가 되고 있기 때문에
사다리를 탄다, 실행, 진행한다 라는 의미를 가진 네이밍 어떨까요?
```suggestion
        Ladder ladder = new Ladder(persons.size(), height.get());
        ladder.climbDown(lineConnectPolicy);
```